tests/expected/intrinsics/ptr_offset_from_unsigned/check_invariant_violation.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
```

Comment:
Please add another description here (this is the case where pointers are given in the wrong order, right?).